examples/with-typescript-graphql/lib/gql/fragment-masking.ts

```diff
@@ -0,0 +1,44 @@
+import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/core'
```

Comment:
This fails linting (`pnpm lint`) could you please address the issue?

Reply:
FIxed